charts/devlake/templates/deployments.yaml

```diff
@@ -56,12 +56,19 @@ spec:
           imagePullPolicy: {{ .Values.ui.image.pullPolicy }}
           ports:
             - containerPort: 4000
+          {{- $httpAuthProbe := (include "devlake.ui.auth.probe" $ | fromYaml | list) }}
           {{- with .Values.ui.livenessProbe }}
           livenessProbe:
+            {{- if not (hasKey .httpGet "httpHeaders") }}
+              {{- $_ := set .httpGet "httpHeaders" ($httpAuthProbe) }}
+            {{- end }}
```

Comment:
action failed, need to add if condition of basic enabled or not
![image](https://github.com/apache/incubator-devlake-helm-chart/assets/135692613/ffce6ce8-cb2a-4d73-8c22-9afa95f79a14)